hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/throttle/CompactionThroughputControllerFactory.java

```diff
@@ -33,10 +36,16 @@
   public static final String HBASE_THROUGHPUT_CONTROLLER_KEY =
       "hbase.regionserver.throughput.controller";
 
+  public static String currentControllerClassName =
```

Comment:
Why static? And do we really need to store the class name here? Why not just call the getClass method on the controller instance?

Reply:
> Why static? And do we really need to store the class name here? Why not just call the getClass method on the controller instance?

Thanks , I think call the getClass is better